### PR TITLE
Add OpenVINO 2022.1 support

### PR DESCRIPTION
**Pull request by szabi-luxonis**
Todo:
-add 2022.1 support for blobconverter
-add tests
-change all blobs to 2022.1

Note that 2021.4 and 2022.1 are intercompatible.

### CODE REVIEW

File: src/utility/Resources.cpp [hunk 1]
```diff
@@ -138,6 +138,7 @@ std::vector<std::uint8_t> Resources::getDeviceFirmware(Device::Config config, da
                 depthaiPatch = resourceMapDevice.at(DEPTHAI_CMD_OPENVINO_2021_3_PATCH_PATH);
                 break;
 
+            case OpenVINO::VERSION_2021_4:
```
Comment (themarpe) [inline src/utility/Resources.cpp:141]: Required?

**Review by themarpe — APPROVED** [review]

File: src/utility/Resources.cpp (hunk 1 above)
Comment (szabi-luxonis) [reply src/utility/Resources.cpp:141]: Just so all cases are handled in switch.

File: .github/workflows/test.workflow.yml [hunk 2]
```diff
@@ -54,7 +54,7 @@ jobs:
     - name: Configure, Build and Test
       if: matrix.os != 'linux'
       run: |
-        cmake -S . -B build -D CMAKE_BUILD_TYPE=Release -D HUNTER_ROOT=$HOME/.hun_${{ matrix.flavor }} -D DEPTHAI_BUILD_EXAMPLES=ON -D DEPTHAI_BUILD_TESTS=ON -D DEPTHAI_TEST_EXAMPLES=ON
+        cmake -S . -B build -D CMAKE_BUILD_TYPE=Release -D HUNTER_ROOT=$HOME/.hun2_${{ matrix.flavor }} -D DEPTHAI_BUILD_EXAMPLES=ON -D DEPTHAI_BUILD_TESTS=ON -D DEPTHAI_TEST_EXAMPLES=ON
```
Comment (themarpe) [inline .github/workflows/test.workflow.yml:57]: Just change the key - this won't be cached anymore

Comment (themarpe) [reply .github/workflows/test.workflow.yml:57]:
```
    steps:
    - name: Cache .hunter folder
      uses: actions/cache@v2
      with:
        path: $HOME/.hun_${{ matrix.flavor }}
        key: hunter-${{ matrix.os }}-${{ matrix.cmake }}-2
    - uses: actions/checkout@v2
with:
```
to 
```
    steps:
    - name: Cache .hunter folder
      uses: actions/cache@v2
      with:
        path: $HOME/.hun_${{ matrix.flavor }}
        key: hunter-${{ matrix.os }}-${{ matrix.cmake }}-2
    - uses: actions/checkout@v2
with:
```

And revert the HUNTER_ROOT path modification

Comment (szabi-luxonis) [reply .github/workflows/test.workflow.yml:57]: That still doesn't work. https://github.com/luxonis/depthai-core/actions/runs/2439316266
Changed the cached path to `hun2`
```
path: $HOME/.hun2_${{ matrix.flavor }}
```

Comment (themarpe) [reply .github/workflows/test.workflow.yml:57]: Sorry - the cached Hunter folder actually remains on selfhosted runners. LGTM for now. Next time, let me know, I"ll purge the cache manually

Comment (szabi-luxonis) [reply .github/workflows/test.workflow.yml:57]: I could have purged too with a temporary commit, I changed the folder keeping in mind older branches that are not up to date etc. so CI won't fail for them

**Pull request merged**